src/store/index.ts

```diff
@@ -22,7 +22,7 @@ const state: any = {
 }
 
 const persistState = createPersistedState({
-    paths: ['user', 'util.productStores'],
+    paths: ['user', 'util.productStores', 'util.statusDesc'],
```

Comment:
Clear status desc data on logout, also check once if all the data is getting cleared on logout.